examples/C++/Keys/keys.cpp

```diff
@@ -221,7 +273,7 @@ void keys()
         }
     }
 
-    eClock::my_sleep(1500);
+    std::this_thread::sleep_for(std::chrono::milliseconds(1500));
```

Comment:
Do we need this `sleep_for` and the ones below? They were not present in #777

Reply:
This sleep_for and the ones below are also present in #777. But in this PR they appear as changes since branch 1.8.x starts from an earlier version of this example in which the calls to the eClock had not yet been deleted.

Reply:
:+1: understood